ballerina-tests/tests/resources/documents/interfaces_implementng_interface_querying_fragement_on_transitive_interface.graphql

```diff
@@ -0,0 +1,33 @@
+query {
+  thisIsAMule: animal(id: 11) {
+    ... on Equine {
+      name,
+      sex,
+      ... getFather
```

Comment:
```suggestion
      ...getFather
```
check other places as as well

Reply:
> check other places as well

I think we need a space here. GraphQL doc/spec follows the same

Reply:
Updated